liquibase-dist/src/main/archive/liquibase

```diff
@@ -33,10 +33,12 @@ if [ -f /usr/bin/cygpath ]; then
   done
 
   # Gather jars from a local liquibase_libs directory
-  for i in liquibase_libs/*.jar; do
-    i=`cygpath --windows "$i"`
-    CP="$CP;$i"
-  done
+  if [[ -f ./liquibase_libs/*.jar ]]
```

Comment:
This is maybe going to fail if there are multiple jar files in liquibase_libs? (based on https://www.rushworth.us/lisa/?p=3315, I haven't tested it).

That page suggests doing an ls to check the count, but does that seem too hacky? Is there a better way to catch that?

(the same change will have to be applied below)